pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -69,6 +69,12 @@
   // Estimated size of offline table
   OFFLINE_TABLE_ESTIMATED_SIZE("OfflineTableEstimatedSize", false),
 
+  // Estimated size of offline table per replica
+  OFFLINE_TABLE_ESTIMATED_SIZE_PER_REPLICA("bytes", false),
+
+  // Estimated size of realtime table per replica
+  REALTIME_TABLE_ESTIMATED_SIZE_PER_REPLICA("bytes", false),
```

Comment:
We don't need to separate metrics for offline and realtime table. Merging them as `TABLE_ESTIMATED_SIZE_PER_REPLICA`